modules/microshift-creating-ostree-iso.adoc

```diff
@@ -0,0 +1,175 @@
+* You have root-user access to your build host.
+* You have the `podman` tool.
+
+.Procedure
+
+. Start an `ostree` container image build by running the following command:
++
+[source,terminal]
+----
+$ BUILDID=$(sudo composer-cli compose start-ostree --ref "rhel/9/$(uname -m)/edge" minimal-microshift edge-container | awk '{print $2}')
+----
++
+This command also returns the identification (ID) of the build for monitoring.
+
+. You can check the status of the build periodically by running the following command:
++
+[source,terminal]
+----
+$ sudo composer-cli compose status
+----
++
+.Example output of a running build
+
+[source,terminal]
+----
+ID                                     Status     Time                      Blueprint            Version   Type               Size
+cc3377ec-4643-4483-b0e7-6b0ad0ae6332   RUNNING    Wed Jun 7 12:26:23 2023   minimal-microshift   0.0.1     edge-container
+----
++
+.Example output of a completed build
+
+[source,terminal]
+----
+ID                                     Status     Time                      Blueprint            Version   Type               Size
+cc3377ec-4643-4483-b0e7-6b0ad0ae6332   FINISHED   Wed Jun 7 12:32:37 2023   minimal-microshift   0.0.1     edge-container
+----
+
+[NOTE]
+====
+You can use the `watch` command to monitor your build if you are familiar with how to start and stop it.
+====
+
+. Download the container image using the ID and get the image ready for use by running the following commands:
++
+[source,terminal]
+----
+$ sudo composer-cli compose image ${BUILDID}
+----
+
+. Change the ownership of the downloaded container image to the current user by running the following command:
++
+[source,terminal]
+----
+$ sudo chown $(whoami). ${BUILDID}-container.tar
+----
+
+. Add current-user read permissions to the image by running the following command:
++
+[source,terminal]
+----
+$ sudo chmod a+r ${BUILDID}-container.tar
+----
+
+. Bootstrap a server on port 8085 for the `ostree` container image to be consumed by the ISO build using the following two steps.
+
+.. Get the IMAGEID result by running the following command:
++
+[source,terminal]
+----
+$ IMAGEID=$(cat < "./${BUILDID}-container.tar" | sudo podman load | grep -o -P '(?<=sha256[@:])[a-z0-9]*')
+----
+
+.. Use the IMAGEID result to execute the podman command step by running the following command:
++
+[source,terminal]
+----
+$ sudo podman run -d --name=minimal-microshift-server -p 8085:8080 ${IMAGEID}
+----
++
+This command also returns the ID of the container saved in the `IMAGEID` variable for monitoring.
+
+. Generate the installer blueprint file by running the following command:
++
+[source,terminal]
+----
+$ cat > microshift-installer.toml <<EOF
+name = "microshift-installer"
+
+description = ""
+version = "0.0.0"
+modules = []
+groups = []
+packages = []
+EOF
+----
+
+[id="microshift-add-blueprint-build-iso_{context}"]
+== Add the blueprint to Image Builder and build the ISO
```

Comment:
This should be a separate module (only one proc per module according to mod docs)